tests/client/amo/containers/TestExtensionsPage.js

```diff
@@ -0,0 +1,26 @@
+import { mapStateToProps, setAddonType } from 'amo/containers/ExtensionsPage';
+
+
+describe('ExtensionsPage mapStateToProps()', () => {
+  it('sets the props', () => {
+    const state = {
+      landing: {
+        featured: { results: 'featured' },
+        highlyRated: { results: 'high' },
+        popular: { results: 'pop' },
+      },
+    };
```

Comment:
Let me know if I'm off base here but I really think setting up raw state is dangerous. It makes it hard to iterate on the structure of our internal state. If you used actions to set up the state then it would mitigate that problem because you'd get test failures in all the right places. Is it too cumbersome to use the actions?